### PR TITLE
Adding ssl support

### PR DESCRIPTION
**Pull request by reinheimer1**

### CODE REVIEW

File: ssl/create_certificates.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+{\rtf1}
```
Comment (ThomasHurek) [inline ssl/create_certificates.sh:1]: This looks weird?

**Review by ThomasHurek — CHANGES_REQUESTED** [review]
Is the create_certificates.sh file correct?

File: ssl/create_certificates.sh (hunk 1 above)
Comment (reinheimer1) [reply ssl/create_certificates.sh]: you are right. looks like the changes did not save correctly. I fixed it now.

**Review by reinheimer1 — COMMENTED** [review]
I solved the findings from Thomas

**Pull request merged**